binding/HarfBuzzSharp.Shared/Face.cs

```diff
@@ -24,11 +41,35 @@ public Face (Blob blob, int index)
 			Handle = HarfBuzzApi.hb_face_create (blob.Handle, index);
 		}
 
+		public Face (GetTableDelegate getTable)
+			: this (getTable, null, null)
+		{
+		}
+
+		public Face (GetTableDelegate getTable, object context)
+			: this (getTable, context, null)
+		{
+		}
+
+		public Face (GetTableDelegate getTable, object context, ReleaseDelegate destroy)
+			: this (IntPtr.Zero)
+		{
+			if (getTable == null)
+				throw new ArgumentNullException (nameof (getTable));
+
+			var ctx = new NativeDelegateContext (
+				new object[] { context, this },
+				new Delegate[] { destroy, getTable });
+			Handle = HarfBuzzApi.hb_face_create_for_tables (table_func, ctx.NativeContext, DestroyFunction.NativePointer);
+		}
+
 		internal Face (IntPtr handle)
 			: base (handle)
 		{
 		}
 
+		public static Face Empty => new Face (HarfBuzzApi.hb_face_get_empty ());
```

Comment:
If we want reference equality for this it should follow the same pattern as `Blob.Empty`